packages/esm-patient-allergies-app/src/allergies/allergies-form/allergy-form.resource.ts

```diff
@@ -2,10 +2,11 @@ import { useMemo } from 'react';
 import useSWR from 'swr';
 import useSWRImmutable from 'swr/immutable';
 import { openmrsFetch, restBaseUrl, useConfig } from '@openmrs/esm-framework';
```

Comment:
```suggestion
import { openmrsFetch, restBaseUrl, useConfig, type OpenmrsResource } from '@openmrs/esm-framework';
```

Consolidate these imports.

Reply:
Done with the changes
will keep in mind next time!